### ISSUE TITLE
Question: 

### ISSUE THREAD

**Issue by BrandonCookeDev** [ripgarpr/garpr#122]
GAwes asked me this the other day:
"I have a question. Does the system have a way of "de-weighting" old results. In FIFA, results are worthless after 4 years, and are weighted 100% in the year they occur, and vary by % in the years in between. Does a win vs GAwes count for as many points in 2018 as it did in 2015 if he's done nothing since?"
@jhertz @jschnei

**Comment by Jamesargy6** [comment]
I've done some investigation into trueskill, and I _think_ that by changing one of the parameters (Tau?), you can introduce more uncertainty into the system, which makes more recent sets affect the player's score more than they currently do. However, this isn't really a solution into making the old sets count less - and more uncertainty in general isn't really a good thing. Trueskill does consider more recent results more important than older, but I don't know exactly how much so, and I'm not sure it can be changed.

**Comment by Jamesargy6** [comment]
http://netprophetblog.blogspot.com/2011/07/experimenting-with-recency.html

Interesting read. I wonder how our numbers would look if we only included, say, the last 6 months worth of results? Old players are still HEAVILY carried by their old results.

**Comment by BrandonCookeDev** [comment]
I mean, theoretically speaking I think maybe a reset at the top of the year would be sufficient, and here's why:

Let's take Wes for example. If he stops playing in 2016, then picks the game back up in 2017 with relatively the same skill, he will likely re-achieve his same position in the rankings. Now let's say he picks it up in 2018 and either he's gotten worse or people have gotten better. His past yearly results aren't calculated, and he starts fresh, achieving his rightful place on the board given the pool of skill he's competing in.

**Comment by Jamesargy6** [comment]
Agreed. This also gives improving players a chance to escape their bad early performance. (Let's take me for example kek)

**Comment by BrandonCookeDev** [comment]
So Wes said he's gonna make a profile later on and participate in this thread, but for the interim I will proxy for him. 

He asks
"
Just read through it. So, a couple of thoughts, because I don't know what's possible or not in the system.
If it's being suggested that a total reset should happen at the mark of the calendar year I disagree with that on a lot of levels. 
Is it possible to have a rolling negation of results at the 365 day  mark. Aka if I beat Baka on July 23 2016 then that win counts for me and that loss counts for him until July 24th 2017 when it's as if it never happened ?
That would be more along how fifas system works which I think is fairly solid. Also, if it's just as simple I would use a little longer than 12 months, maybe 15
"

**Comment by Jamesargy6** [comment]
Yeah I think a rolling period is much more practical, and yeah also totally doable.

**Comment by BrandonCookeDev** [comment]
I also really want to hear what @jschnei and @jessemtso think about it
